onnxruntime/test/platform/apple/apple_package_test/Podfile.template

```diff
@@ -1,14 +1,34 @@
-platform :ios, '13.0'
+def should_exclude_macos_target
+    if '@C_POD_NAME@' != 'onnxruntime-mobile-c'
```

Comment:
should it be: pod name == 'onnxruntime-mobile-c' means exclude the macos target?

nit: instead of `should_exclude_macos_target` and the conditional `unless should_exclude_macos_target` with a double negative, could we just have `include_macos_target`?
```ruby
include_macos_target = ('@C_POD_NAME@' != 'onnxruntime-mobile-c')
if include_macos_target
  ...
end
```

Reply:
you are right. double negative is a bit confusing. and forgot to change the condition when adding training configs.

fixed.